code/index.html

```diff
@@ -1,22 +1,225 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kaisei+Opti:wght@400;500;700&family=Open+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/df7601e6e4.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+    <section class="full__container">
+      <header class="main__header">
+        <img src="images/header-logo.svg"/>
+      </header>
+      <nav class="my__nav">
+        <a href="#">Startpage</a>
+        <a href="#">Big news</a>
+        <a href="#">Small news</a>
+        <a href="#">Contact us</a>
+      </nav>
+      <section class="main__section">
+        <div>
+          <img src="images/kitty-hero@2x.jpg" />
+          <div>
+            <h2 class="big-news__header">
+              Welcome to the very purrrent cat(news)meows
+            </h2>
+            <p>
+              It's 3am, time to create some chaos please let me outside pouty
+              face yay! wait, it's cold out please let me inside pouty face oh,
+              thank you rub against mommy's leg oh it looks so nice out, please
+              let me outside again the neighbor cat was mean to me please let me
+              back inside flop over check cat door for ambush 10 times before
+              coming in Plan steps for world domination meow.<br /><br />
+              Be superior. I like big cats and i can not lie so you're just
+              gonna scroll by without saying meowdy? hate dogs. The dog smells
+              bad do doodoo in the litter-box, clickityclack on the piano, be
+              frumpygrumpy cat playing a fiddle in hey diddle diddle? or lay on
+              arms while you're using the keyboard for run as fast as i can into
+              another room for no reason chew the plant. Plays league of
+              legends.<br />
+              <br />
+              Scratch my tummy actually i hate you now fight me make muffins at
+              four in the morning wake up owner meeeeeeooww scratch at legs and
+              beg for food then cry and yowl until they wake up at two pm jump
+              on window and sleep while observing the bootyful cat next door
+              that u really like but who already has a boyfriend end up making
+              babies with her and let her move in but waffles but meoooow or
+              human is washing you why halp oh the horror flee scratch hiss
+              bite. Cattt catt cattty cat being a cat.
+            </p>
+          </div>
+        </div>
+        <div>
+          <img src="images/kitty1@2x.jpg" />
+          <h3>This is Herald</h3>
+          <p>
+            A 2 year old playful male. He likes meowing, food and late night
+            zoomies.
+          </p>
+          <span>Adopt me!</span>
+        </div>
+        <div class="yellow-box">
```

Comment:
According to the code guidelines, we should focus on the intention not the implementation when writing class names. E.g. "adopt-box".